### PR TITLE
Improve the application reload

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/build [hunk 1]
```diff
@@ -164,8 +168,10 @@ def main() -> None:
             if not service.startswith("redis")
         ]
 
-        run(args, ["docker-compose", "rm", "--stop", "--force"] + services)
-        run(args, ["docker-compose", "up", "-d"])
+    if args.fast_reload is not None:
```
Comment (ochriste) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/build:171]: didnt you remove fast_reload from the allowed argparse parameters ?

Comment (ochriste) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/build:171]: maybe you mean "if args.reload" ?

Comment (ochriste) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/build:171]: or is this some backward compatible hack ?

**Review by arnaud-morvan — APPROVED** [review]
This is exactly what I'm used to do:

```
./build --no-pull --service=config env.xxx \
&& docker-compose rm -f -v config \
&& docker-compose up -d --force-recreate myservice \
&& docker-compose logs -f myservice
```

**Pull request merged**